### ISSUE TITLE
setPedAnimation should accept the "blend" parameter when stopping an animation

### ISSUE THREAD

**Issue by Einheit-101** [multitheftauto/mtasa-blue#881]
**Description**
Currently it is impossible to "smoothly" stop an animation, using setPedAnimation while not specifying the type stops the current animation immediately.
In that case, the "blend" parameter is ignored. It would be better if the "blend" parameter could be used to smoothly stop the current animation.

**Additional context**
Related to #275

**Comment by ghost** [comment]
Correct! `blendTime` is only used for blending the new animation with the previous one. It's not meant to affect the animation if it's being destroyed. If we allow this parameter to affect the animation when it's being stopped, then this would mean we're changing the default behavior. Not everybody would want it this way. I believe that it would be better to use a new parameter.

**Comment by qaisjp** [comment]
@saml1er we could check to see if blendTime was actually provided instead of just the default. If it's explicitly provided, then it could be used.

I doubt (many) people are actually doing `setPedAnimation(ped, nil, nil, -1, true, true, true, true, 250)` instead of ``setPedAnimation(ped, nil, nil)` or `setPedAnimation(ped); even if being done programmatically.

Scripters might find it different to have different behaviour when explicitly providing the arg vs. the default, though. 🤔

**Comment by Einheit-101** [comment]
qaisjp is correct. Providing the argument should be reason enough.

**Comment by qaisjp** [comment]
Not too sure if it would be confusing or not.

We'd basically be saying "default blend is 0 if you are cancelling the animation" (which is what it is now... except blend is always 0 when cancelling)
